LambdaEngine/Source/Game/ECS/Systems/Rendering/AnimationSystem.cpp

```diff
@@ -126,15 +130,27 @@ namespace LambdaEngine
 		// Animation system has its own clock to keep track of time
 		m_Clock.Tick();
 
+		TArray<uint32> jobIndices;
 		for (Entity entity : m_AnimationEntities.GetIDs())
 		{
 			AnimationComponent& animation = pAnimationComponents->GetData(entity);
 			if (!animation.IsPaused)
 			{
-				Animate(animation);
+				std::function<void()> func = [&]
```

Comment:
I think `[&]` captures a reference of everything reachable within the scope. In this case, you only need `this` and `&animation`.